compiler/src/codegen/compcore.re

```diff
@@ -3550,7 +3566,13 @@ let compile_wasm_module = (~env=?, ~name=?, prog) => {
       default_features;
     };
   let _ = Module.set_features(wasm_mod, features);
-  let _ = Settings.set_low_memory_unused(true);
+  // we set low_memory_unused := true iff the user has not specified a memory base.
```

Comment:
```suggestion
  // we set low_memory_unused := true if the user has not specified a memory base.
```
nit

Reply:
Not a typo, though?

Reply:
Had to google `iff` but it seems that it is a bi-directional if, which isn't true in this scenario, right? If something in the program somehow `set_low_memory_unused`, you wouldn't enable the memory base.

Reply:
Yes, `iff` is a bidirectional if ("if and only if"). It is true in this scenario, since we are the ones setting it right here. On the next line, we set the value of `low_memory_unused` to whatever the value of `Option.is_none(memory_base^)` is, meaning that `low_memory_unused` will be `true` if and only if `--memory-base` was not passed.

Reply:
```suggestion
  // we set low_memory_unused := true if and only if the user has not specified a memory base.
```
I shouldn't have to google to understand comments 😛